### PR TITLE
v24.1.3 release notes

### PR DESCRIPTION
**Pull request by kathancox**
Fixes REL-1097

### CODE REVIEW

File: src/current/_includes/releases/v24.2/v24.2.0-alpha.1.md [hunk 1]
```diff
@@ -176,6 +176,7 @@ This release includes 1234 merged PRs by 97 authors.
 [#124681]: https://github.com/cockroachdb/cockroach/pull/124681
```
Comment (kathancox) [inline src/current/_includes/releases/v24.2/v24.2.0-alpha.1.md:176]: Some links were broken in this alpha.1 and I realized they were missing.

File: src/current/_includes/releases/v24.1/v24.1.3.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+<h3 id="v24-1-3-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where collection of [debug information]({% link v24.1/cockroach-debug-zip.md %}) for very long-running [jobs]({% link v24.1/show-jobs.md %}) could use excessive space in the `job_info` system table and cause some interactions with the jobs system to become slow. [#126122][#126122]
+- Fixed a bug where a change to a [user-defined type]({% link v24.1/create-type.md %}) could cause queries against tables using that type to fail with an error: `histogram.go:694: span must be fully contained in the bucket`. The change to the user-defined type could come directly from an [`ALTER TYPE`]({% link v24.1/alter-type.md %}) statement, or indirectly from an [`ALTER DATABASE ADD REGION`]({% link v24.1/alter-database.md %}#add-region) or [`DROP REGION`]({% link v24.1/alter-database.md %}#drop-region) statement (which implicitly change the [`crdb_internal_region`]({% link v24.1/alter-table.md %}#set-the-table-locality-to-regional-by-row) type). This bug has existed since UDTs were introduced in v20.2. [#125800][#125800]
```
Comment (rmloveland) [inline src/current/_includes/releases/v24.1/v24.1.3.md:20]: suggest adding "user-defined-type (UDT)" so the later mention of "UDT" is defined

File: src/current/_includes/releases/v24.1/v24.1.3.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+- Fixed a bug in which constant `LIKE` patterns containing certain sequences of backslashes did not become constrained scans. This bug has been present since v21.1.13 when support for building constrained scans from `LIKE` patterns containing backslashes was added. [#125539][#125539]
```
Comment (rmloveland) [inline src/current/_includes/releases/v24.1/v24.1.3.md:21]: mention of 'LIKE' could link to https://www.cockroachlabs.com/docs/v24.1/sql-feature-support.html#scalar-expressions-and-boolean-formulas

File: src/current/_includes/releases/v24.1/v24.1.3.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+- The `sql_sequence_cached_node` value of the [`serial_normalization` setting]({% link v24.1/serial.md %}#modes-of-operation) was not correctly formatted. This could lead to errors while connecting to CockroachDB if the default value of `serial_normalization` was set to `serial_normalization`. The formatting bug was fixed, which also fixes the errors when connecting. [#127675][#127675]
+
+<h3 id="v24-1-3-performance-improvements">Performance improvements</h3>
+
+- Starting a `cockroach` process will no longer flush buffered [logs]({% link v24.1/logging-overview.md %}) to configured [logging sinks]({% link v24.1/configure-logs.md %}#configure-log-sinks) unless the process is running under `systemd`, where cockroach runs with the `NOTIFY_SOCKET` environment variable. [#126305][#126305]
+- [Schema changes]({% link v24.1/online-schema-changes.md %}) that cause a data backfill, such as adding a non-nullable column or changing the primary key, will now split and scatter the temporary indexes used to perform the change. This reduces the chance of causing a write hotspot that can slow down foreground traffic. [#126684][#126684]
```
Comment (rmloveland) [inline src/current/_includes/releases/v24.1/v24.1.3.md:38]: 'write hotspot' could link to https://www.cockroachlabs.com/docs/v24.1/performance-best-practices-overview#hot-spots

File: src/current/_includes/releases/v24.2/v24.2.0-alpha.1.md (hunk 1 above)
Comment (rmloveland) [reply src/current/_includes/releases/v24.2/v24.2.0-alpha.1.md:176]: is this due to a bug in the `release-notes.py` script? if so would you please consider filing a DOC issue with the 'Product Area = Edu Eng - Docs Toolchain' and ping Eeshan ?

Comment (kathancox) [reply src/current/_includes/releases/v24.2/v24.2.0-alpha.1.md:176]: I don't think so, I actually think I did these notes and just did a poor job of removing things!

**Pull request merged**